app-creator/style.css

```diff
@@ -25,9 +25,12 @@ p {
   --accent-color: #202124;
```

Comment:
Since the list of css properties is getting longer here, would you mind sorting them alphabetically? See go/htmlcssstyle#Declaration_Order